docs/API Overview/pis.md

```diff
@@ -1,56 +1,160 @@
+
 - Domestic Scheduled Payments
+
 - Domestic Standing Orders
 
+  
+
 The payment request from TPP must have the creditor information.
+
 ## Overview
-The following apply to all domestic payment consents:
+
+  
+
+### The following apply to **all payment consents (domestic and international):**
+
+  
 
 ### `InstructedAmount/Amount`
+
 - There is no MAX `InstructedAmount/Amount` mandated by Stripe API.
--  Stripe suggest PISP notify the PSU that the same limits apply as in their Stripe app. It is possible from time to time that `domestic-payment-consents` is authorised, but the payment initiation fails due to account limits.
 
-### `InstructedAmount/Currency`
-- `InstructedAmount/Currency` must be `GBP`
+- Stripe suggest PISP notify the PSU that the same limits apply as in their Stripe app. It is possible from time to time that the payment consent is authorised, but the payment initiation fails due to account limits.
+
+  
 
 ### `RemittanceInformation/Reference`
+
 `RemittanceInformation/Reference` is a mandatory field and must adhere to the following:
+
 - Valid characters:
-  - A-Z
-  - a-z
-  - 0-9
-  - "space"
-  - The characters "&", "-", ".", "/"
+
+- A-Z
+
+- a-z
+
+- 0-9
+
+- "space"
+
+- The characters "&", "-", ".", "/"
+
+  
 
 - Contiguous characters – user enters 6 or more valid characters but without contiguous string of at least 6 alphanumeric characters
+
 - Must contain a contiguous string of at least 6 alphanumeric characters
+
 - Homogeneous string – user enters 6 or more valid characters (including valid non-alphanumeric characters) - After stripping out non-alphanumeric characters the resulting string cannot consist of all the same character
 
+  
+
 The PISP may also opt to populate reference field on behalf of the PSU
 
+  
+
+### `SupplementaryData` requirements
+
+Stripe's Open Banking API requires you to send additional data in the SupplementaryData field to ensure a successful payment:
+
+-  `SupplementaryData.CreditorAccount.Email`: Must contain the recipient's email address
+
+-  `SupplementaryData.CreditorAccount.EntityType`: Must be either `individual` or `company` depending on the type of the recipient
+
+### The following apply to **all domestic payment consents:**
+
 ### `CreditorAccount` requirements
 
-`CreditorAccount` supports either `UK.OBIE.SortCodeAccountNumber` or `UK.OBIE.Wallet` for the `Account.SchemeName` parameter. Providing any other value will return an error.
-- `UK.OBIE.Wallet`
-  - `CreditorAccount.Identification`: Must be the recipient account ID (e.g. `acct_test_123`)
-  - `SupplementaryData.CreditorAccount.Destination`: Optional if the recipient has a default destination, otherwise the destination ID (e.g. `usba_test_123`)
+`CreditorAccount` supports either `UK.OBIE.SortCodeAccountNumber` or `UK.OBIE.Wallet` or `UK.OBIE.IBAN` (only for international payments to French recipients) for the `Account.SchemeName` parameter. Providing any other value will return an error.
```

Comment:
Why add this here considering the title of the section is "The following apply to **all domestic payment consents"